client/src/components/App.js

```diff
@@ -11,24 +11,43 @@ import ProfileInfoEdit from './profileInfoEdit/ProfileInfoEdit';
 
 import Register from './auth/Register';
 import Login from './auth/Login';
-import Header from './layout/Header'
+import Header from './layout/Header';
+import Sidebar from './layout/Sidebar';
+import { Container, Col, Row } from 'react-bootstrap';
+
+import './layout/Layout.css'
 
 import PublicationPage from './publications/PublicationPage';
+import { Fragment } from 'react';
 
 const App = () => {
+    const urls={
+        dashboard: "/dashboard",
+        profile: "/dashboard/profile"
+    }
     return (
-
         <BrowserRouter>
-            <Header />
             <Switch>
-                <Route path="/" exact component={Home}/>
-                <Route path="/auth" exact component={Auth}/>
-                <Route path="/publications/team/:teamId" exact component={PublicationPage}/>
-                <Route path="/register" exact component={Register}/>
-                <Route path="/login" exact component={Login}/>
-                <Route path="/dashboard" exact component={Dashboard}/>
-                <Route path="/dashboard/profile" exact component={ProfileInfoEdit} />
-            </Switch>
+                <Route path="/" exact component={Home} />
+                <Route path="/auth" exact component={Auth} />
+                <Route path="/register" exact component={Register} />
+                <Route path="/login" exact component={Login} />
+                <Fragment>
+                    <Header title={"Researchify"} urls={urls}/>      
+                    <Container fluid>
+                        <Row>
+                        <Col className="sidebar-wrapper" md={2} lg={2} xl={1}>
+                            <Sidebar />
+                        </Col>
+                        <Col className="page-content-wrapper" md={10} lg={10} xl={11}>
+                            <Route path="/publications/team/:teamId" exact component={PublicationPage}/>
+                            <Route path={urls.dashboard} exact component={Dashboard} />
+                            <Route path="/dashboard/profile" exact component={ProfileInfoEdit} />
+                        </Col>
+                        </Row>
+                    </Container>
+                </Fragment>                
```

Comment:
Is there a way we can refactor this block to a separate component? How about encapsulating it all in `Header` itself?
I think we should keep the root `App` component clean and avoid defining UI here unless we don't have a choice.

Reply:
+1

Reply:
Will try to refactor this part in next sprint.